### PR TITLE
Clean up error handling in move-mv-llvm-compiler

### PR DESCRIPTION
**Pull request by brson**
Handle errors instead of calling `expect` in main.

### CODE REVIEW

**Review by aditya-solana — APPROVED** [review]

**Comment by aditya-solana** [comment]
LGTM

**Comment by brson** [comment]
I can't merge this myself, if that is what you are waiting for. No permissions.

**Pull request merged**
